src/osemgrep/configuring/Semgrep_settings.ml

```diff
@@ -0,0 +1,97 @@
+type t = {
```

Comment:
Maybe add a TODO comment like
(* we should use ATD to specify this settings file *)

Reply:
done in [9306c04](https://github.com/returntocorp/semgrep/pull/7637/commits/9306c04c582e7d6724195fea1e0b102f0dc9335a)